src/database/models/exile.py

```diff
@@ -1,9 +1,17 @@
 class Exile(object):
     def __init__(
-        self, exile_id, user_id, reason, exile_status, start_timestamp, end_timestamp
+        self,
+        exile_id,
+        user_id,
+        discord_id,
+        reason,
+        exile_status,
+        start_timestamp,
+        end_timestamp,
     ):
         self.exile_id = exile_id
         self.user_id = user_id
+        self.discord_id = (discord_id,)
```

Comment:
I don't think you want this to be a tuple, I think it should just be `discord_id`

Reply:
![image](https://github.com/user-attachments/assets/e72b1529-ff07-4ae7-b8eb-1bb4b8c05d07)
output looked like this when it wasnt

Reply:
nevermind i see the error.